tests/ptf/ptf_runner.py

```diff
@@ -286,6 +286,9 @@ def main():
     parser.add_argument('--skip-test',
                         help='Skip test execution (useful to perform only pipeline configuration)',
                         action="store_true", default=False)
+    parser.add_argument('--skip-launch',
+                        help='Skip switch launch (requires that switch be started manually beforehand)',
```

Comment:
I would rename the argument to `--skip-bmv2-start` and I would clarify in the help string that this option applies only when using BMv2 as a target. As when using, for example, Tofino, we already assume that the switch agent instance is already running.

Reply:
Done.  I just pushed the change with your suggestion.  Thanks.